### PR TITLE
Fix build due to fenced frame spec update

### PR DESCRIPTION
**Pull request by linnan-github**
Will prepare a PR to move the monkeypatch to fenced frame spec soon.

***
<a href="https://pr-preview.s3.amazonaws.com/linnan-github/conversion-measurement-api/pull/1051.html" title="Last updated on Sep 30, 2023, 8:50 PM UTC (21b2b47)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/1051/ce437a6...linnan-github:21b2b47.html" title="Last updated on Sep 30, 2023, 8:50 PM UTC (21b2b47)">Diff</a>

### CODE REVIEW

**Comment by linnan-github** [comment]
@domfarolino, could you PTAL, thanks!

File: index.bs [hunk 1]
```diff
@@ -532,9 +532,9 @@ Add the step:
 
 # Fenced Frame monkeypatches # {#fenced-frame-monkeypatches}
 
-Add the following items to [=fencedframetype/pending event=]:
+Add the following items to [=fencedframetype/destination enum event=]:
 
-<dl dfn-for="pending event">
+<dl dfn-for="destination enum event">
```
Comment (domfarolino) [inline index.bs:537]: Hmm, why are these two lines necessary to fix the build? A "pending event" still exists in the FF spec: https://wicg.github.io/fenced-frame/#fencedframetype-pending-event.

File: index.bs [hunk 2]
```diff
@@ -546,98 +546,57 @@ Modify {{Fence/reportEvent()}} as follows:
...
-
-<div algorithm="report-an-event-monkeypatch">
+Before the step
 
-Add a [=Document=] parameter called |document|.
-
-After the step
-
-> If |reporting map|[|destination|] does not [=map/exist=], return.
+> [=list/For each=] |destination| of <var ignore=''>event</var>'s {{FenceEvent/destination}}:
```
Comment (domfarolino) [inline index.bs:551]: In what algorithm? You should probably link to it in the "Before the step in the [....] algorithm" above, to make this much more clear. Or maybe I'm missing something? Basically I wouldn't reference an unattached step like this...

Comment (domfarolino) [inline index.bs:551]: You don't need `ignore=''` here. You just need `ignore`.

File: index.bs [hunk 3]
```diff
@@ -546,98 +546,57 @@ Modify {{Fence/reportEvent()}} as follows:
...
 
 <div algorithm="send-a-beacon-monkeypatch">
 
-Add a [=boolean=] parameter called |attributionReportingEnabled| and an [=origin=]
-parameter called |contextOrigin|.
-
 After the step
 
-> Let <var ignore>destination url</var> be <var ignore>destination map</var>[<var ignore>eventType</var>].
+> Let <var ignore=''>destination url</var> be an empty [=string=].
```
Comment (domfarolino) [inline index.bs:581]: Same as above...

File: index.bs (hunk 1 above)
Comment (linnan-github) [reply index.bs:537]: Yeah, but I think "destination enum event" is where these two properties are needed.

File: index.bs (hunk 2 above)
Comment (linnan-github) [reply index.bs]: Thanks, it's in the reportEvent() algorithm (please expand the unchanged text above), but added a few more steps to be clearer.

File: index.bs (hunk 1 above)
Comment (domfarolino) [reply index.bs:537]: OK, so this isn't related to the "fixing a broken build", this is just a general spec change that's included here?

File: index.bs (hunk 2 above)
Comment (domfarolino) [reply index.bs]: Ah, sorry I missed that.

**Review by domfarolino — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (linnan-github) [reply index.bs:537]: Thanks Dom. The build error is due to accessing the eventType/eventData field of pending event, but I refactored with the latest fenced frame spec.

**Review by apasel422 — APPROVED** [review]

**Pull request merged**